### ISSUE TITLE
Auto lr return 0 steps due to diverging loss even though I  pass a early_stop_threshold to be None

### ISSUE THREAD

**Issue by Rami-Ismael** [PyTorchLightning/pytorch-lightning#11375]
## 🐛 Bug

Auto lr return 0 steps due to diverging loss even though I  pass a early_stop_threshold to be None

### Environment

- PyTorch Lightning Version (e.g., 1.5.0):1.5.3
- PyTorch Version (e.g., 1.10):1.10.1
- Python version (e.g., 3.9): 3.7.12
- OS (e.g., Linux):Linux
- CUDA/cuDNN version:10.2
- GPU models and configuration: 1 
- How you installed PyTorch (`conda`, `pip`, source): pip install
- If compiling from source, the output of `torch.__config__.show()`:
- Any other relevant information: This is running on google colab

### Additional context

cc @akihironitta @borda @rohitgr7

**Comment by carmocca** [comment]
Hi!

Can you provide more information about what you are trying to do? And perhaps share your code

**Comment by Rami-Ismael** [comment]
https://colab.research.google.com/drive/1RYmuCFgbWagDLgZglx4kN7sKIPFDyugb?usp=sharing.
I want to use the auto lr from pytorch lightning. However, the tuner stop because of diverging loss. However, the threshold was set to None. My expectation was the the tuner will continue find the a good learning rate even loss is divergining because the value for threshold is set to none

**Comment by rohitgr7** [comment]
hey @Rami-Ismael,
I tried the [bug_report_model](https://colab.research.google.com/github/PyTorchLightning/pytorch-lightning/blob/master/pl_examples/bug_report/bug_report_model.ipynb) and it seems to be working as expected. The example you shared is too big to check the underlying issue. Is it possible to get a minimal version with the same behavior?

**Comment by Rami-Ismael** [comment]
The underlying issue in method objective_1.  The problem
`
    # Run learning rate finder
    lr_finder = trainer.tuner.lr_find(model = model   ,early_stop_threshold=None)` it should not give me a `LR finder stopped early after 0 steps due to diverging loss.
Restoring states from the checkpoint path at /content/lr_find_temp_model_a9ba9fb8-b858-494e-8a1d-754743a4ccc5.ckpt`
